pkg/apis/pulumi/v1/stack_types.go

```diff
@@ -18,6 +18,9 @@ type StackStatus struct {
 	// ObservedGeneration records the value of .meta.generation at the point the controller last processed this object
```

Comment:
Adding additional status fields for our controller to keep track of reconcile progression.